components/uploadFormHook.tsx

```diff
@@ -37,91 +37,86 @@ export const UploadFormHook: FC = () => {
-            <label />
-          </div>
-
-          <div className="mb-5">
-            <p>所属会社</p>
-            {/* フォームのサイズを調整すべき */}
-            {/* <input {...register('company')} placeholder="株式会社オープンアップグループ" required /> */}
+          </label>
+          <label className="block">
+            <span className="after:content-['*'] after:ml-0.5 after:text-red-500">
+              所属会社
+            </span>
             <input
-              className="w-80"
+              className="block w-80"
               {...register('company')}
               placeholder="オープンアップグループ"
               required={true}
             />
-            <label />
-          </div>
-
-          <div className="mb-5">
-            <p>社員番号</p>
+          </label>
+          <label className="block">
+            <span className="after:content-['*'] after:ml-0.5 after:text-red-500">
+              社員番号
+            </span>
             <input
-              className="w-80"
+              className="block w-80"
               {...register('employeeId')}
               placeholder="123456"
               required={true}
             />
-            <label />
-          </div>
-
-          <div className="mb-5">
-            <p>連絡可能な個人電話番号</p>
+          </label>
+          <label className="block">
+            <span className="after:content-['*'] after:ml-0.5 after:text-red-500">
+              連絡可能な個人電話番号
+            </span>
             <input
-              className="w-80"
+              className="block w-80"
               {...register('phone')}
               placeholder="090-1234-5678"
               required={true}
             />
-            <label />
-          </div>
-
-          <div className="mb-5">
-            <p>メールアドレス</p>
+          </label>
+          <label className="block">
+            <span className="after:content-['*'] after:ml-0.5 after:text-red-500">
+              メールアドレス
+            </span>
             <input
-              className="w-80"
+              className="block w-80"
               {...register('mail')}
               placeholder="example@mail.com"
               required={true}
             />
-            <label />
-          </div>
-
-          <div className="mb-5">
-            <p>個人情報提供に同意いただけますか?</p>
-            <p>
-              <input type="radio" {...register('agreement')} required={true} />
-              <label />
-              同意する
-            </p>
-          </div>
-
-          <div>
-            <ImagePreview />
-            <ImageInput
-              id="image"
-              fileInputRef={{ current: null }}
-              {...register('image')}
-            />
-          </div>
+          </label>
+          <label className="block">
+            <span className="after:content-['*'] after:ml-0.5 after:text-red-500">
+              個人情報提供への同意
+            </span>
+            <br />
+            同意する
+            <input type="radio" {...register('agreement')} required={true} />
+          </label>
```

Comment:
I forget to change type from "radio" to "checkbox".
```suggestion
            <input type="checkbox" {...register('agreement')} required={true} />
```

Reply:
I've updated `radio` to `checkbox`.